resolwe/flow/executors/docker.py

```diff
@@ -2,17 +2,30 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import os
+import random
 import shlex
 import subprocess
 
+from django.conf import settings
+
 from .local import FlowExecutor as LocalFlowExecutor
 
 
 class FlowExecutor(LocalFlowExecutor):
     def start(self):
+        container_image = settings.FLOW_EXECUTOR['CONTAINER_IMAGE']
+
+        if self.data_id != 'no_data_id':
+            container_name = 'resolwe_{}'.format(self.data_id)
+        else:
+            # set random container name for tests
+            rand_int = random.randint(1000, 9999)
```

Comment:
Docker container is not removed if you interrupt tests. So this is added just as convenience that you don't have to manually remove containers after each interrupted test. This will never happen on Jenkins. So I don't think that UUIDs are needed here.